crates/rolldown_binding/src/utils/normalize_binding_options.rs

```diff
@@ -96,6 +96,15 @@ pub fn normalize_binding_options(
     module_types = Some(tmp);
   }
 
+  let mut globals = None;
+  if let Some(raw) = output_options.globals {
+    let mut tmp = HashMap::with_capacity(raw.len());
+    for (k, v) in raw {
```

Comment:
If change `BundlerOptions#globals` to `: Option<FxHashMap<String, String>>`, Here could be using `  let globals = output_options.globals.map(|raw| FxHashMap::from_iter(raw.into_iter()));
`.

Reply:
I don't know why it reports error both in this method or `FxHashMap::from(...)`. 

```
error[E0308]: mismatched types
   --> crates/rolldown_binding/src/utils/normalize_binding_options.rs:142:5
    |
142 |     globals,
    |     ^^^^^^^ expected `Option<HashMap<String, String>>`, found `Option<HashMap<String, String, ...>>`
    |
    = note: expected enum `std::option::Option<std::collections::HashMap<_, _, RandomState>>`
               found enum `std::option::Option<std::collections::HashMap<_, _, FxBuildHasher>>`

For more information about this error, try `rustc --explain E0308`.
error: could not compile `rolldown_binding` (lib) due to 1 previous error
error: Recipe `check-rust` failed on line 50 with exit code 101
error: Recipe `check` failed on line 56 with exit code 101
(
```

Reply:
Look like not change `BundlerOptions#globals` to `Option<FxHashMap<String, String>>`

Reply:
Solved